README.md

```diff
@@ -76,42 +73,41 @@ keyboardToolbar.addBackButton(title: "Back")
 keyboardToolbar.addNextButton(title: "Next")
 keyboardToolbar.addFlexibleSpace()
 keyboardToolbar.addSystemDoneButton()
-}
 ```
 Check out `KeyboardToolbar` for other button adding options.
 
 
 ### KeyboardNavigator - when using a KeyboardToolbar
-Create a KeyboardToolbar, configuring it with back/next/done buttons as appropriate. Then, create a KeyboardNavigator, passing in your text inputs and toolbar. The order of the text inputs determines the navigation order for traversing from one to the next. Optionally, implement `KeyboardNavigatorDelegate` to receive call backs when tapping "Back", "Next", and "Done" in your `KeyboardToolbar`. **Please see the example project for tips when using `UITextViews`.**
+Create a `KeyboardToolbar`, configuring it with back/next/done buttons as appropriate. Then, create a `KeyboardNavigator`, passing in your text inputs and toolbar. The order of the text inputs determines the navigation order for traversing from one to the next. Optionally, implement `KeyboardNavigatorDelegate` to receive call backs when tapping "Back", "Next", and "Done" in your `KeyboardToolbar`. **Please see the example project for tips when using `UITextViews`.**
```

Comment:
Now that it's no longer necessary for the user to manually set `currentTextInputIndex`, are the tips in the sample project still relevant?